pkg/model/config/config.go

```diff
@@ -136,9 +205,13 @@ func (c Config) resourceAPIVersionCompatible(verType, version string) bool {
 		var currVersion string
 		switch verType {
 		case "crd":
-			currVersion = res.CRDVersion
+			if res.API != nil {
+				currVersion = res.API.Version
+			}
 		case "webhook":
-			currVersion = res.WebhookVersion
+			if res.Webhooks != nil {
+				currVersion = res.Webhooks.Version
```

Comment:
Same here
```suggestion
				currVersion = res.Webhooks.WebhookVersion
```

Reply:
Same as in https://github.com/kubernetes-sigs/kubebuilder/pull/1869#discussion_r540080054